macro/vignettes/fluxmovement.Rmd

```diff
@@ -0,0 +1,146 @@
+---
+title: "Flux Movement"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Flux Movement}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(macro)
+```
+
+## Stochastic Model
+
+### Master Equations
+
+In this document we'll talk about the stochastic flux model. In this model people move between various locations (patches), where the probability of movement depends on where the person is now and where they might go. Because we consider continuous time models, the probability of moving from patch $i$ to $j$ ($i\rightarrow j$) in an infinitesimal time interval $\delta t$ is $P(i\rightarrow j) = f_{ij}\delta t + \mathcal{O}(\delta t)$.
+
+For verification of the simulation we will want to compare the time-averaged simulation results to an analytic stationary distribution. Let's consider a model with 2 patches and 2 people. Because in the flux model people do not have homes (they just move between patches according to the rate parameters $f_{ij}$ wherever they are), this model has 3 states.
+
+$$
+S_{1} = (N_{1}=1,N_{2}=1) \\
+S_{2} = (N_{1}=2,N_{2}=0) \\
+S_{3} = (N_{1}=0,N_{2}=2)
+$$
+
+The total state space is $\mathcal{S}:=\{S_{1},S_{2},S_{3}\}$, and each time an event occurs, the state changes. $S_{1}$ is the state where one person is in patch 1 and one person is in patch 2. $S_{2}$ is the state where both people are in patch 1. $S_{3}$, likewise, is where both people are in patch 3. In order to get a stationary probability distribution over these 3 states, we need to work out the master equations. These are a set of ordinary differential equations (ODE) which describe how the probability flux moves between states as time goes on. Put in terms of an example, if at $t=0$ we knew with absolute certainty that both people were in patch 1 ($S_{2}$ gets probability 1 at time 0), we could solve the master equations to tell us how probability flows out of and back into that state (as well as between all other states) as time goes on.
```

Comment:
There's a sentence in this paragraph where you say what S1, S2 and S3 are. I might have gotten the equations immediately if that liner were before the equations, right after "this model has 3 states."

Reply:
Good point.